src/PrestaShopBundle/Resources/views/Admin/Improve/International/Tax/Blocks/form.html.twig

```diff
@@ -0,0 +1,73 @@
+{#**
+ * 2007-2019 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% import '@PrestaShop/Admin/macros.html.twig' as ps %}
+
+{{ form_start(taxForm) }}
+<div class="card">
+  <div class="card-header">
+    {{ 'Taxes'|trans({}, 'Admin.Global') }}
+  </div>
+
+  <div class="card-block row">
+    <div class="card-text">
+      {% set nameHint %}
+        {{ 'Tax name to display in carts and on invoices (e.g. "VAT").'|trans({}, 'Admin.International.Help') }}
+        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}
+      {% endset %}
+
+      {{ ps.form_group_row(taxForm.name, {}, {
+        'label': 'Name'|trans({}, 'Admin.Global'),
+        'help': nameHint,
+      }) }}
+
+      {% set rateHint %}
+        {{ 'Format: XX.XX or XX.XXX (e.g. 19.60 or 13.925)'|trans({}, 'Admin.International.Help') }}
+      {% endset %}
+
+      {{ ps.form_group_row(taxForm.rate, {}, {
+        'label': 'Rate'|trans({}, 'Admin.International.Feature'),
+        'help': rateHint,
+      }) }}
+
+      {{ ps.form_group_row(taxForm.is_enabled, {}, {
+        'label': 'Enable'|trans({}, 'Admin.Actions')
+      }) }}
+    </div>
+  </div>
+
+  <div class="card-footer">
+    <div class="d-inline-flex">
+      <a href="{{ path('admin_taxes_index') }}" class="btn btn-outline-secondary">
+        {{ 'Cancel'|trans({}, 'Admin.Actions') }}
+      </a>
+    </div>
+    <div class="d-inline-flex float-right">
+      <button class="btn btn-primary">{{ 'Save'|trans({}, 'Admin.Actions') }}</button>
+    </div>
+  </div>
+
+</div>
+{{ form_end(taxForm) }}
```

Comment:
```suggestion
  {{ form_rest(taxForm) }}
  {{ form_end(taxForm) }}
{% endblock %}
```

- add a `form_rest` to allow prestashop developers to easily add new fields to the form without having to modify the template
- add the endblock matching the block I mentioned above